docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md

```diff
@@ -254,6 +269,22 @@ It's very important that you always request an access token to your own services
 
 You can determine if the signed-in user has an Exchange Online account or on-premise Exchange account by using the [Office.UserProfile.accountType](/javascript/api/outlook/office.userprofile) property. If the account type property value is **enterprise**, then the mailbox is on an on-premises Exchange server. Note that volume-licensed perpetual Outlook 2016 doesn’t support the **accountType** property. To work around this, call the [ResolveNames](/exchange/client-developer/web-service-reference/resolvenames-operation) operation in Exchange Web Service (EWS) in the Exchange on-premise server to get the recipient types.
 
+## How do I deploy my add-in to Microsoft AppSource
+
+If you are publishing a new add-in to Microsoft AppSource, it will need to go through a certification process. For more information, see [Publish your Office Add-in to Microsoft AppSource](../publish/publish-office-add-ins-to-appsource.md). If you are updating the manifest of an add-in that is already published in Microsoft AppSource, you need to go through the certification process again. You can update the add-in's source code on your web server any time without a need to go through the certification process.
+
+If you're add-in uses SSO through NAA, your add-in must be in compliance with the following publishing guidelines.
+
+- [1000.3 Authentication options](/legal/marketplace/certification-policies#10003-authentication-options)
+- [1120.3 Functionality](/legal/marketplace/certification-policies#11203-functionality)
+
+Be sure to handle admin consent properly. See [Publish an add-in that requires admin consent for Microsoft Graph scopes](../publish/publish-nested-app-auth-add-in.md)
+
+Also important details here for new or updated add-ins. See Make your solutions available in Microsoft AppSource and within Office - Marketplace publisher | Microsoft Learn
+If you update your add-in (change the manifest) you need to go through the certification process again.
```

Comment:
Does this line need a link? It seems something is missing here and on the next line. They seem out of place.